### ISSUE TITLE
tox -e docs fails

### ISSUE THREAD

**Issue by mrunge** [python-babel/django-babel#30]
```
tox -edocs
GLOB sdist-make: /home/mrunge/work/django-babel/setup.py
docs inst-nodeps: /home/mrunge/work/django-babel/.tox/dist/django-babel-0.4.0.zip
docs installed: alabaster==0.7.7,Babel==2.2.0,Django==1.9.2,django-babel==0.4.0,docutils==0.12,Jinja2==2.8,MarkupSafe==0.23,Pygments==2.1.1,pytz==2015.7,six==1.10.0,snowballstemmer==1.2.1,Sphinx==1.3.5,sphinx-rtd-theme==0.1.9
docs runtests: PYTHONHASHSEED='2878308748'
docs runtests: commands[0] | sphinx-build -W -b html -d /home/mrunge/work/django-babel/.tox/docs/tmp/doctrees docs /home/mrunge/work/django-babel/.tox/docs/tmp/_build/html
Running Sphinx v1.3.5
making output directory...
loading pickled environment... not yet created
loading intersphinx inventory from https://docs.python.org/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 5 source files that are out of date
updating environment: 5 added, 0 changed, 0 removed
reading sources... [ 20%] extract
reading sources... [ 40%] index
reading sources... [ 60%] management-commands
reading sources... [ 80%] middleware
reading sources... [100%] template-tags


Warning, treated as error:
/home/mrunge/work/django-babel/docs/template-tags.rst:4: WARNING: autodoc: failed to import module u'django_babel.templatetags.babel'; the following exception was raised:
Traceback (most recent call last):
  File "/home/mrunge/work/django-babel/.tox/docs/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 385, in import_object
    __import__(self.modname)
  File "/home/mrunge/work/django-babel/.tox/docs/lib/python2.7/site-packages/django_babel/templatetags/babel.py", line 3, in <module>
    from babel import support as babel_support
ImportError: cannot import name support

ERROR: InvocationError: '/home/mrunge/work/django-babel/.tox/docs/bin/sphinx-build -W -b html -d /home/mrunge/work/django-babel/.tox/docs/tmp/doctrees docs /home/mrunge/work/django-babel/.tox/docs/tmp/_build/html'

```

**Comment by graingert** [comment]
@mrunge does this happen on travis?

**Comment by mrunge** [comment]
@graingert this happens on a local environment, I did not look at travis at all. How is that hooked into here?

**Comment by graingert** [comment]
@mrunge if you make a PR travis runs a build

**Comment by Itxaka** [comment]
Travis tox for docs runs under python3.4: https://travis-ci.org/python-babel/django-babel/jobs/102118827

But the tox.ini file does not specify any python version. Running this locally I can trigger this issue on python2.7. Forcing the docs env to python3.4 (basepython=python3.4) makes the error dissapear.

**Comment by graingert** [comment]
looks like the links are broken too. Should be https://babel.readthedocs.org/en/latest/api/core.html#babel.core.Locale

**Comment by Itxaka** [comment]
@graingert yes, happening on local. Im guessing that before, the links redirected properly (http://babel.pocoo.org/en/docs/api/core/#babel.core.Locale -> http://babel.pocoo.org/en/latest/api/core.html) because there was only one version of the docs? Now there are 2 (latest, stable) so the redirection is not automatic?

**Comment by graingert** [comment]
I think this is fixed in 00e92e7
